Src/Model/Csv.php

```diff
@@ -1,38 +1,54 @@
 <?php
-
 namespace Aot\Model;
 
 
 class Csv
 {
     private $path;
 
-    public function __construct()
+    public function __construct(){
+        $this->path = ROOT . 'Source/testtakers.csv';
+    }
+
+    /**
+     * define the method
+     * @param $method
+     * @param null $id
+     * @return array
+     * @throws \Exception
+     */
+    public function csvMethod($method, $id = null)
     {
-        $this->path = ROOT . '/Source/testtakers.csv';
+        if ($method == 'show' && $id) {
```

Comment:
If the behavior depends on the HTTP parameters, it's part of the routing. You should concentrate the responsibility of routing on the router